gallery/visualtests/nxCombobox.js

```diff
@@ -266,17 +266,216 @@ describe('NxCombobox', function() {
+      expect(await isVisible(dropdownMenu)).toBe(false);
+    });
+
+    it('reopens dropdown after selection when input is refocused', async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          dropdownMenuSelector = `${basicExampleSelector} .nx-dropdown-menu`,
+          buttonSelector = `${dropdownMenuSelector} .nx-dropdown-button:first-child`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      let [dropdownMenu, firstOptBtn] = await waitAndGetElements(dropdownMenuSelector, buttonSelector);
+
+      await firstOptBtn.click();
+      expect(await isVisible(dropdownMenu)).toBe(false);
+
+      await blurElement(input);
+      expect(await isFocused(input)).toBe(false);
+
+      await input.focus();
+      [dropdownMenu] = await waitAndGetElements(dropdownMenuSelector);
+      expect(await isVisible(dropdownMenu)).toBe(true);
+    });
+
+    it('reopens dropdown after selection when the input is clicked on', async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          dropdownMenuSelector = `${basicExampleSelector} .nx-dropdown-menu`,
+          buttonSelector = `${dropdownMenuSelector} .nx-dropdown-button:first-child`,
+          example = '#nx-combobox-basic-example',
+          [input, basicExample] = await waitAndGetElements(inputSelector, example);
+
+      await input.click();
+      let [dropdownMenu, firstOptBtn] = await waitAndGetElements(dropdownMenuSelector, buttonSelector);
+
+      await firstOptBtn.click();
+      expect(await isVisible(dropdownMenu)).toBe(false);
+
+      // check the dropdown opens with the input having never lost focus
+      expect(await isFocused(input)).toBe(true);
+      await input.click();
+      [dropdownMenu] = await waitAndGetElements(dropdownMenuSelector);
+      expect(await isVisible(dropdownMenu)).toBe(true);
+
+      // check the dropdown opens after refocusing the input
+      await basicExample.click();
+      expect(await isFocused(input)).toBe(false);
+      expect(await isVisible(dropdownMenu)).toBe(false);
+      await input.click();
+      [dropdownMenu] = await waitAndGetElements(dropdownMenuSelector);
+      expect(await isFocused(input)).toBe(true);
+      expect(await isVisible(dropdownMenu)).toBe(true);
+    });
+  });
+
+  describe('dropdown display behavior with backend query', function() {
+    it('shows the dropdown when focused and the user has typed a value', async function() {
+      const inputSelector = `${backendExampleSelector} .nx-combobox__input input`,
+          dropdownMenuSelector = `${backendExampleSelector} .nx-dropdown-menu`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('i');
+
+      const [dropdownMenu] = await waitAndGetElements(dropdownMenuSelector);
+      expect(await isVisible(dropdownMenu)).toBe(true);
```

Comment:
The key difference from the predetermined list case is that the dropdown should not open before the user types a value.  This test should check that

Reply:
> The key difference from the predetermined list case is that the dropdown should not open before the user types a value. This test should check that

added here: [46dd648](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/46dd648cb7c5a57ca0e0597587b9e59c7afccd09)